src/main/java/ReviewResponse.java

```diff
@@ -1,10 +1,13 @@
 import lombok.Data;
 import lombok.Getter;
+import lombok.Setter;
 import lombok.ToString;
 
 import java.util.List;
 
 @ToString
+@Setter
```

Comment:
Using Lombok's @Setter annotation can expose all fields for modification externally, breaking encapsulation. (Severity: medium)